### PR TITLE
Clean up drivetrain

### PR DESCRIPTION
**Pull request by gnawinganimal**
I made some changes to this, just to clean it up.

### CODE REVIEW

File: src/main/kotlin/org/sert2521/bunnybots2019/drivetrain/Drivetrain.kt [hunk 1]
```diff
@@ -7,8 +7,8 @@ import org.sert2521.bunnybots2019.rightFront
 import org.sert2521.sertain.motors.MotorController
 import org.sert2521.sertain.subsystems.Subsystem
 
-class Drivetrain : Subsystem("Drivetrain", ::driveTrain) {
-    private val right = MotorController(rightFront, rightBack) {
+class Drivetrain : Subsystem("Drivetrain", ::controlDrivetrain) {
+    private val right = MotorController(rightFront, rightBack){
```
Comment (andrewda) [inline src/main/kotlin/org/sert2521/bunnybots2019/drivetrain/Drivetrain.kt]:
```suggestion
    private val right = MotorController(rightFront, rightBack) {
```

File: src/main/kotlin/org/sert2521/bunnybots2019/utils/DrivetrainUtils.kt [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package org.sert2521.bunnybots2019.utils
 
+fun Double.deadband(range: Double) =
+        if (this < range && this > -range) 0.0 else this
+
+
```
Comment (andrewda) [inline src/main/kotlin/org/sert2521/bunnybots2019/utils/DrivetrainUtils.kt]:
```suggestion
```

**Review by andrewda — COMMENTED** [review]
;)

**Review by will-hou — APPROVED** [review]
Should be up do date with `development` now. Thanks for the styling changes, Andrew!

**Review by andrewda — APPROVED** [review]
❤️

**Pull request merged**